### PR TITLE
Grammar check

### PR DESCRIPTION
**Pull request by Akshobhya1234**
I used Grammarly and latex language support to fix grammar in this PR.

### CODE REVIEW

File: AlgebraSurvey.tex [hunk 1]
```diff
@@ -41,12 +41,12 @@ \section{Proof Assistant Systems}
 discussed in the previous chapter. 
 
 \subsection{Coq}
-Coq \cite{Paulin-Mohring2012} is a theorem-proving system that is written in the
 OCaml programming language. It is designed to assist in the formalization of
-mathematical theorems by using constructive and higher-order logic.  It was
-first released in 1989 and is one of the most widely used proof assistant
-systems to define mathematical definitions, and theory and to write proofs. The
-Coq specification language is called Gallina. Users can write functions and
+Coq \cite{Paulin-Mohring2012} is a theorem-proving system that is written in the
```
Comment (JacquesCarette) [inline AlgebraSurvey.tex:44]: this now makes no sense as the 2nd line has been flipped with the first! You should be more careful

File: ProblemInProgramAlgebra.tex [hunk 2]
```diff
@@ -111,9 +111,9 @@ \section{Equivalent But Structurally Different}
 \begin{equation}
 y ∙ a = b
 \end{equation}
-Another definition of quasigroup is given as type a (2,2,2) algebra in which a
-set $Q$ and binary operations $∙, \backslash, /$. In Chapter 5, we define
-quasigroup with three binary operations that satisfy the below identities.
+Another definition of quasigroup is given as type a (2,2,2). In Chapter 5, we
+defined quasigroup with three binary operations ($∙, \backslash, /$) that
+satisfy the below identities.
```
Comment (JacquesCarette) [inline ProblemInProgramAlgebra.tex]: identities below

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: Agda.tex [hunk 3]
```diff
@@ -243,29 +243,29 @@ \section{Type Levels In Agda}
...
+for some arbitrary \inline{ℓ}. To solve this problem, Agda provides type
+\inline{Level}. The type \inline{Set ℓ} represents the type of all types at
+level \inline{ℓ}. For example, \inline{Set 0} represents \inline{Set₀},
 \inline{Set 1} represents \inline{Set₁}, and so on. This type helps us to define
 equality generalized to an arbitrary level.
 
 \section{Equality}
 \label{equality}
-In Chapter 2, when defining theory, we say that equation is of the form $t_1 =
+In Chapter 2, when defining theory, we said the equation is of the form $t_1 =
```
Comment (JacquesCarette) [inline Agda.tex]: 'said that an'

File: Agda.tex [hunk 4]
```diff
@@ -463,14 +463,14 @@ \section{Structure Definition}
...
-    rightId : ∀ {x} → op x e ≡ x  
+    rightId : ∀ {x} → op x e ≡ x
 \end{minted}
 
-In the above example, we see that \inline{IsMonoid} structure has a parameter
-\inline{A : Set} with fields \inline{e} - the identity element and \inline{op} -
-the binary operation. We also give the laws of monoid structure as its field.
-Another way to define a monoid structure is to parameterize the binary operation
-and the identity element.
+In the above example, we \inline{IsMonoid} structure has a parameter \inline{A :
```
Comment (JacquesCarette) [inline Agda.tex]: that 'see that' was needed for the sentence to make sense

File: Agda.tex [hunk 5]
```diff
@@ -805,8 +806,8 @@ \section{Product Algebra}
 groups, rings, or vector spaces.
 
 The products of various structures are defined under the module
-\mintinline[breaklines,breakafter=.]{Agda}{Algebra.Construct.DirectProducts} in Agda standard library. The product
-of magma structure is defined as:
+\mintinline[breaklines,breakafter=.]{Agda}{Algebra.Construct.DirectProducts} in
+Agda standard library. The product of magma structure is defined as:
```
Comment (JacquesCarette) [inline Agda.tex]: 'the' Agda standart library

File: Agda.tex [hunk 6]
```diff
@@ -837,7 +838,7 @@ \section{Equational Proofs In Agda}
 proposition using equational reasoning within Agda's type system
 \cite{murray2022constructive}. 
 
-In section \ref{types}, we have seen how to define natural numbers and addition
+In section \ref{types}, we saw how to define the natural number and addition
```
Comment (JacquesCarette) [inline Agda.tex]: Actually, the original was correct (have seen -> saw is fine), but the other part's meaning has been broken by this change.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: AlgebraSurvey.tex (hunk 1 above)
Comment (Akshobhya1234) [reply AlgebraSurvey.tex:44]: Ah, I select sentences while reading. When I move the mouse (by mistake), vscode will shift the selected part. This change was not intentional.

File: Agda.tex (hunk 5 above)
Comment (Akshobhya1234) [reply Agda.tex]: I have fixed the comments. Please review.

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**